docs/source/_ext/probnum-config-options.py

```diff
@@ -0,0 +1,80 @@
+from docutils import nodes
+from docutils.statemachine import StringList
+from sphinx.util.docutils import SphinxDirective, switch_source_input
+
+import probnum as pn
+
+
+class ProbNumConfigOptions(SphinxDirective):
+    def run(self):
+        table = nodes.table(
+            "",
+            classes=[
+                "longtable",
+                "colwidths-auto",
+                # "colwidths-given",
```

Comment:
is this on purpose?

Reply:
Yup. In `docutils.nodes`, basically nothing is documented and it took some digging to find out how to adjust column widths automatically. In case we want to switch to statically sized columns, I added the respective command in order to save us the pain in the future

Reply:
Do you think it makes sense to put a quick comment into the source here that explains exactly this issue? If not, that is okay as well.

Reply:
In my opinion, the keywords are sufficiently expressive, and a comment is not necessary. But I can add one, if you prefer :)

Reply:
I think it will do the job without (especially if I would be the only one to whom this ocmment wasnt obvious :))